content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,89 @@
-=======================
-What is a Putaway Rule?
-=======================
+=============
+Putaway Rules
+=============
+
+Putaway is the process of receiving shipments and routing the products in them to the appropriate
+storage locations. This process can be accomplished seamlessly in Odoo using *putaway rules*.
+
+The rules detail how products move into and throughout specified locations in the warehouse.
+They are triggered by the arrival of shipments in a certain location. The automatically generated
+rules coordinate efficient inter-warehouse transfers.
 
-A good warehouse implementation takes care that products automatically move to their appropriate
-destination location. To make that process easier, Odoo uses *Putaway rules*. But what is a putaway
-rule? Putaway is the process of taking products off the receiving shipments and directly putting
-them into the most appropriate location.
+If, for example, a warehouse contains multiple volatile substances that should not come into
+contact, it is crucial that these substances not be stored in close proximity to one another. Using
+putaway rules, it is possible to ensure that such products are stored in different locations.
 
-If, for example, a warehouse contains volatile substances, it is important to make sure that certain
-products are not stored close to each other because of a potential chemical reaction. That’s where
-putaway rules intervene, to avoid storing products wrongly.
+.. seealso::
+   - :ref:`Routes-- how products move<inventory/routes/use_routes/push>`
+   - :ref:`Warehouse locations<inventory/warehouse_location/definition>`
 
 Configuration
```

Comment:
```suggestion
Configuration
```
What are we configuring? The heading could be something more specific e.g., `Configure putaway rules` or `Configure warehouse settings`